metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatUpdateTablePreEvent.java

```diff
@@ -51,8 +52,8 @@ public MetacatUpdateTablePreEvent(
         @Nonnull @NonNull final QualifiedName name,
         @Nonnull @NonNull final MetacatRequestContext requestContext,
         @Nonnull @NonNull final Object source,
-        @Nonnull @NonNull final TableDto oldTable,
-        @Nonnull @NonNull final TableDto currentTable
+        @Nullable final TableDto oldTable,
```

Comment:
Now that these are nullable you basically need to go to every place these events are used and check for null too...